modelci/common.py

```diff
@@ -0,0 +1,238 @@
+#
```

Comment:
Is it necessary to create a `common.py` file? We've already have a `utils` package. If you feel your functions is very general that may be reused somewhere else, put them in the `utils` package. Otherwise, you should move them within the TRT conversion related packages.